website/content/api-docs/secret/pki.mdx

```diff
@@ -16,1178 +16,1109 @@ update your API calls accordingly.
 
-This is an unauthenticated endpoint.
+It is _strongly_ encouraged to limit the scope of CAs within a mount and not
+to mix different types of CAs (roots and intermediates).
 
-| Method | Path            |
-| :----- | :-------------- |
-| `GET`  | `/pki/ca(/pem)` |
+~> **Note**: Some functionality will not work if a default issuer is not
+   configured. Vault automatically selects the default issuer from the
+   current issuing certificate on migration from an older Vault version.
 
-### Sample Request
+## Accessing Authority Information
 
-```shell-session
-$ curl \
-    http://127.0.0.1:8200/v1/pki/ca/pem
-```
+All consumers of the PKI Secrets Engine mount point will have access to the
+following unauthenticated APIs, useful for reading information about the
+certificate authority in this mount point.
 
-### Sample Response
+This includes information about CA certificates, their chains, and signed
+CRLs, containing an encoded list of revoked certificates previously
+issued by this authority.
 
-```text
-<binary DER-encoded certificate>
-```
+However, two endpoints (for listing all certificates issued by this authority
+and listing all issuers in this mount) are authenticated, though not generally
+considered privileged information.
 
-## Read CA Certificate Chain
+### List Issuers
 
-This endpoint retrieves the CA certificate chain, including the CA _in PEM
-format_. This is a bare endpoint that does not return a standard Vault data
-structure and cannot be read by the Vault CLI; use `/pki/cert` for that.
-Additionally, note that this doesn't include the root authority and so may
-return empty data depending on configuration; use `/pki/cert/ca_chain`'s
-`ca_chain` JSON data field for the entire chain including issuing authority.
+This endpoint returns a list of issuers currently provisioned in this mount.
+The response includes both the issuer's identifier as well as the name chosen
+by the operators; either can be used to refer to the issuer later.
 
-This is an unauthenticated endpoint.
+~> Note: This endpoint is authenticated.
 
-| Method | Path            |
-| :----- | :-------------- |
-| `GET`  | `/pki/ca_chain` |
+| Method | Path           |
+| :----- | :------------- |
+| `LIST` | `/pki/issuers` |
 
-### Sample Request
+#### Sample Request
 
 ```shell-session
 $ curl \
-    http://127.0.0.1:8200/v1/pki/ca_chain
+    --header "X-Vault-Token: ..." \
+    --request LIST \
+    http://127.0.0.1:8200/v1/pki/issuers
 ```
 
-### Sample Response
+#### Sample Response
 
-```text
-<PEM-encoded certificate chain>
+```json
+{
+  "data": {
+    "key_info": {
+      "1ae8ce9d-2f70-0761-a465-8c9840a247a2": {
+        "issuer_name": "imported-root"
+      },
+      "3dc79a5a-7a6c-70e2-1123-94b88557ba12": {
+        "issuer_name": "root-x1"
+      }
+    },
+    "keys": [
+      "1ae8ce9d-2f70-0761-a465-8c9840a247a2",
+      "3dc79a5a-7a6c-70e2-1123-94b88557ba12"
+    ]
+  }
+}
 ```
 
-## Read Certificate
+<a name="read-ca-certificate"></a>
```

Comment:
Can we remove this, or what does this do?  Nothing points to `#read-ca-certificate` anymore

Reply:
This allows bookmarks to the (existing, soon to be old) docs persist and link to the right thing in the new updated format. :-)